ReactiveCocoa/Swift/Action.swift

```diff
@@ -134,7 +134,7 @@ public final class Action<Input, Output, Error: ErrorType> {
 public final class CocoaAction: NSObject {
 	/// The selector that a caller should invoke upon a CocoaAction in order to
 	/// execute it.
-	public static let selector: Selector = "execute:"
+	public static let selector: Selector = #selector(CocoaAction.execute(_:))
```

Comment:
> The current release does build with Swift 2.2, so I'm :+1: to be 2.2+ only in the next release.

Yeah, that was my thinking too :)

As for the version, I'm torn. In the end it doesn't really matter. In general I'd love if as a community we could all agree on a pattern to follow with this so that we're all on the same page. I'd argue that following the same minor/major change semantics as Swift would be the clearest.